.github/workflows/static.yaml

```diff
@@ -1,7 +1,7 @@
 #
 # This file is part of Astarte.
 #
-# Copyright 2021 Ispirata Srl
+# Copyright 2023 Ispirata Srl
```

Comment:
Is it ok to keep `Ispirata` as in the other files in this repo, or for the new file we should  change to  `SECO Mind`?
@bettio